docs/features/software-catalog/life-of-an-entity.md

```diff
@@ -183,6 +183,59 @@ The stitching is currently a fixed process, that cannot be modified or extended.
 This means that any modifications you want to make on the final result, has to
 happen during ingestion or processing.
 
-## Deletion
-
-> TODO
+## Orphaning
+
+As mentioned earlier, entities internally form a graph. The edges go from
+processed parent entities, to child entities emitted while processing the
+parent.
+
+The processing loop runs continuously, so these edges are reconsidered over
+time. If processing a parent entity no longer emits a given child entity, then
+that former edge is severed. If that child has no other edges pointing at it
+either, it becomes _orphaned_. The end result is as follows:
+
+- The stitching process injects a `backstage.io/orphan: 'true'` annotation on
+  the child entity.
+- The child entity is _not_ removed from the catalog, but stays around until
+  explicitly deleted via the catalog API, or "reclaimed" by the original parent
+  or another parent starting to reference it.
+- The child entity stops being subjected to regular processing, frozen in time.
+- The catalog page in Backstage for the child entity detects the new annotation
+  and informs users about the orphan status.
+
+Orphaning can occur in several different scenarios. One common cause is that the
+end user edited a corresponding catalog catalog-info YAML file removing the
+entity's entry, or in the case of a `Location` entity, removed the target row
+pointing to the file containing the file. Another common cause is large batch
+processors such as the ones that crawl through remote systems looking for
+entities, no longer finding something that it used to find before. Maybe the
```

Comment:
I guess this is true right now, but may no longer be true in the future with migration to entity providers. Perhaps we can use a more specific example that we don't anticipate will change?